exo-run/src/service-runner.ls

```diff
@@ -76,6 +76,8 @@ class ServiceRunner extends EventEmitter
     dependencies = []
     for dependency-name, dependency-config of @service-config.dependencies or {}
       container-name = "#{@config.app-name}-#{dependency-name}"
+      if dependency-config?.version?
+        dependency-name += ":#{dependency-config.version}"
```

Comment:
I believe this could be simplified to 

`dependency-name += ":{that}"`

as `that` would be referring to `dependency-config.version` already